### PR TITLE
Bump all generated-file and reviewed-file DFXML version references

### PR DESCRIPTION
**Pull request by ajnelson-nist**
This PR clears TODOs that noted certain branches needed to merge in to the schema.  Given the schema testing being used in file generation--parse round-tripping, all branches are assumed to have been merged.

### CODE REVIEW

**Comment by ajnelson-nist** [comment]
Absent feedback, I will merge this change Tuesday morning.

File: tests/misc_object_tests/FileObject_test.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def test_empty_file_object() -> None:
 
 
 def test_blank_file_object_filename() -> None:
-    dobj = Objects.DFXMLObject(version="1.2.0")
+    dobj = Objects.DFXMLObject(version="2.0.0-beta.0")
```
Comment (simsong) [inline tests/misc_object_tests/FileObject_test.py:46]: Should we have all of these strings in a single location?

**Review by simsong — CHANGES_REQUESTED** [review]
I recommend putting all of the version numbers in a single place, perhaps a file called `constants.py` in a class called `C`. Like this:
https://github.com/Plant-Tracer/webapp/blob/main/constants.py

it will make it easier to maintain...

File: tests/misc_object_tests/FileObject_test.py (hunk 1 above)
Comment (ajnelson-nist) [reply tests/misc_object_tests/FileObject_test.py:46]: I'll put the value into `/dfxml/__init__.py`, [next to the XMLNS constants](https://github.com/dfxml-working-group/dfxml_python/blob/68fef71b5c432e5f7c2fbbe84903ff94bd29cd6e/dfxml/__init__.py#L64-L66).

I think I'll also just have the `version` keyword argument default to that constant string.

Comment (simsong) [reply tests/misc_object_tests/FileObject_test.py:46]: I want a single constant string... Should I wait off on merging this?

Comment (ajnelson-nist) [reply tests/misc_object_tests/FileObject_test.py:46]: Yes, please.  I was mid-stream on something else, will handle now.

Comment (ajnelson-nist) [reply tests/misc_object_tests/FileObject_test.py]: This is addressed now.  If you have no other feedback, I'm fine with this being merged at your convenience.

**Review by simsong — APPROVED** [review]
yay!

**Pull request merged**